src/routes/route.py

```diff
@@ -0,0 +1,32 @@
+class Route:
+    def __init__(self, path: list[str], num_people_on_route: int) -> None:
```

Comment:
You've already defined a type for `path`s in the previous PR. If we use the type here, and change the type of routes or vertices in the future, MyPy will automatically complain here if they don't fit.